server/controllers/dashboardController.ts

```diff
@@ -0,0 +1,18 @@
+import type { Request, RequestHandler, Response } from 'express'
+import { jwtDecode } from 'jwt-decode'
+import { sectionsForUser } from '../utils/userUtils'
+
+export default class DashboardController {
+  index(): RequestHandler {
+    return (_req: Request, res: Response) => {
+      const { authorities: roles = [] } = jwtDecode(res.locals.user.token) as { authorities?: string[] }
```

Comment:
ah this is cool, thought we would have to make another API call!

Reply:
Quite minor but the test coverage report is saying line 8 isn't covered by tests...

Reply:
credit to @edavey for spotting that we could use what was in the token!

Reply:
is it important/valuable to mock jwtDecode to get that line covered, do you think?

Reply:
yes, I would say that it's useful documentation: for the unit tests to describe how the roles are extracted from `res.local.user.token` by `jwtDecode()`

Reply:
ok, I have added a new test - let me know if you're happy with it!